fineract-provider/src/test/java/org/apache/fineract/cob/loan/FetchAndLockLoanStepDefinitions.java

```diff
@@ -1,178 +0,0 @@
-/**
```

Comment:
Why did you remove the whole test?

Reply:
This was testing LoanLockTasklet execute method. LoanLockTasklet was no more required as it was applying softlock for loans.